shared/src/persistence/awsDynamoPersistence.test.js

```diff
@@ -61,6 +66,42 @@ describe('awsDynamoPersistence', function() {
     client.updateConsistent.restore();
   });
 
+  describe('getRecordViaMapping', () => {
+    it('should map respondent to active case', async () => {
+      const key = '5678';
+      const type = '234';
+      const isVersioned = undefined;
+      await getRecordViaMapping({
+        applicationContext,
+        key,
+        type,
+        isVersioned,
+      });
+      /*
+      expect(client.query.getCall(0).args[0].Item.pk).to.equal(
+        `${respondentId}|activeCase`,
+      );
+      */
+      expect(client.get.getCall(0).args[0].Key.sk).not.to.equal('0');
```

Comment:
Should this check that it _does_ equal something (`5678`, I think?) rather than not equaling `0`?